ui/apps/ui/src/app/collections/data/all/adapter.data.ts

```diff
@@ -165,9 +165,6 @@ export const allCollectionsAdapter: IAdapter = {
       id: string;
     }
   ): IResult => ({
-    isSortCollectionScopeOff: true,
```

Comment:
I am trying to recall what was the purpose of this variable. Perhaps to completely turn off sort by functionality for e.g. providers? Nonetheless, good job for deleting a deprecated variable 👏

Reply:
thanks - it just wasn't used anywhere so no point in keeping it